packages/map-template/src/atoms/legendHeightState.js

```diff
@@ -0,0 +1,8 @@
+import { atom } from 'recoil';
+
+const legendHeightState = atom({
```

Comment:
We use it in the LegendDialog component only. Consider using a useState hook instead.

Reply:
✅